### ISSUE TITLE
phppgadmin.sourceforge.net is out of date

### ISSUE THREAD

**Issue by adaugherity** [phppgadmin/phppgadmin#96]
There is no mention of versions newer than 5.6 on the [official website](http://phppgadmin.sourceforge.net/doku.php).  I only happened to discover new versions are available by running `locate` on my mirror server and seeing 7.12.1 in my Ubuntu repo!  (I run the tarball, and not even on Ubuntu, so this was a lucky find). This was such a version skip I investigated packages.ubuntu.com to see if it was even the same package (apparently it is) and found this GitHub repo.

Please update the official website, or if that website will no longer be updated, post a big deprecation notice on it.  (In that case, numerous references to that URL in the code itself would need changing, e.g [in intro.php](https://github.com/phppgadmin/phppgadmin/blob/46944bf9bff6642eb8eb26d4fceb211a65deede3/intro.php#L61) (the home screen of the app), `lang/*.php`, etc.)

**Comment by by73c0d3** [comment]
I'd like to second this - we ran on out of date versions for quite a long time, as googling for phppgadmin always leads to the outdated sourceforge homepage. 

It really should explain that there is a new home of phppgadmin.
Alternatively - a hook to push the latest version to sourceforge.

**Comment by xzilla** [comment]
Ok, finally figured out how to update the Sourceforge web page. While the project page already had a pointer to github, the website listed above didn't, I've set it to hard redirect to github. I've also updated the wiki project page in Sourceforge with a note about github releases, and added a news item saying the project is on github, and flagged the project as moved in the admin tool. Hopefully this will be enough to steer people to the right place.
